vm/wasm/api.go

```diff
@@ -1467,7 +1467,7 @@ func tcCallContract(eng *vm.Engine, index int64, args []uint64) (uint64, error)
 
 	vmem := runningFrame.VM.VMemory()
 	appName, err := vmem.GetString(args[0])
-	if err != nil {
+	if err != nil || !common.IsHexAddress(string(appName)) {
 		return 0, err
 	}
```

Comment:
if !common.IsHexAddress(string(appName)),
return a err like fmt.Errorf("%s is not a address", appName)